backend/tests/services/external_events/test_reference.py

```
@@ -0,0 +1,543 @@
+    )
+    assert binding is not None
+    assert binding.provider == "gitlab"
+    assert binding.opaque_ref == "acme/app!7"
+    assert binding.metadata_json["workflow_node_id"] == "wait-1"
+
+
+def test_binding_prefers_requirement_id_when_multiple_wait_nodes_share_provider(
+    test_db: Session,
+    test_user: User,
+) -> None:
+    project, item = _issue(test_db, test_user)
```

Ruff (0.16.1)

[warning] Unpacked variable `project` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)